src/ontology/mondo-edit.obo

```diff
@@ -212070,8 +212070,8 @@ property_value: curated_content_resource "https://www.malacards.org/card/immunod
 [Term]
 id: MONDO:0009465
 name: multiple intestinal atresia
-def: "A rare form of intestinal atresia characterized by the presence of numerous atresic segments in the small bowel (duodenum) or large bowel and leading to symptoms of intestinal obstruction: vomiting, abdominal bloating and inability to pass meconium in newborns." [https://rarediseases.info.nih.gov/diseases/3013/intestinal-atresia-multiple]
-comment: This term's classification was reviewed in the context of the Strategic Refinement project (2023) and was determined to be excluded from the 'immune system disorder' (MONDO:0005046) ontology branch (https://orcid.org/0000-0001-8216-5084)
+def: "A rare form of intestinal atresia characterized by the presence of numerous atresic segments in the small bowel (duodenum) or large bowel and leading to symptoms of intestinal obstruction: vomiting, abdominal bloating and inability to pass meconium in newborns." [Orphanet:2300, PMID:18302063, PMID:33674188]
+comment: This term's classification was reviewed in the context of the Strategic Refinement project (2023) and was determined to be excluded from the 'immune system disorder' (MONDO:0005046) ontology branch (https://orcid.org/0000-0001-8216-5084). It is generally accepted that this term refers to non-hereditary multiple intestinal atresia PMID:33674188. If there is a need for a specific term for non-hereditary multiple intestinal atresia, a new term can be added to Mondo to represent this concept.
```

Comment:
The comment is misleading as it suggests that this Mondo term refers to the non-hereditary version of the disease. 

I suggest updating this comment to:
This term represents both "hereditary" and "non-hereditary" multiple intestinal atresia. It is generally accepted to not distinguish the non-hereditary concept, and use the term "multiple intestinal atresia" to refer to non-hereditary multiple intestinal atresia (PMID:33674188). Since this distinction is not necessary, we have not created a specific term for "non-hereditary multiple intestinal atresia". However, if the need arises, a new term can be added to Mondo to represent this concept